ecommerce/templates/master.html

```diff
@@ -52,8 +54,8 @@
                                     <li><a href="{% url 'categorie' 'accessories' %}">Accessories</a></li>
                                 </ul>
                             </li>
-                            <li><a href="contact.html">Contact Us</a></li>
-                            <li><a href="about.html">About Us</a></li>
+                            <li><a href="/contact_us">Contact Us</a></li>
+                            <li><a href="/about_us">About Us</a></li>
```

Comment:
you should add the name of the actual url specified in ecommerce/urls.py

Reply:
Solved!